main.tf

```diff
@@ -1,3 +1,16 @@
+provider "aws" {
+
+}
+
+provider "aws" {
+  alias   = "alternate"
+  profile = "awsalternate"
+}
+
+provider "aws" {
+  alias = "ns"
+}
```

Comment:
These provider blocks need to be removed from here. This kind of setup is valid for the _test_ configs, where needed, but it is not valid for any reusable modules.